tests/testRandom1.py

```diff
@@ -91,8 +91,8 @@ def testPolicy(self):
         pol = pexPolicy.Policy()
         seed = getSeed()
         pol.set("rngSeed", str(seed))
-        pol.set("rngAlgorithm", afwMath.Random.getAlgorithmNames()[afwMath.Random.RANLXD2])
-        r1 = afwMath.Random(afwMath.Random.RANLXD2, seed)
+        pol.set("rngAlgorithm", afwMath.Random.getAlgorithmNames()[int(afwMath.Random.Algorithm.RANLXD2)])
+        r1 = afwMath.Random(afwMath.Random.Algorithm.RANLXD2, seed)
```

Comment:
Ok. We should probably add this to the list of things to make sure downstream users know about the new pipeline.